docs/source/advanced_features/cloud_storage.rst

```diff
@@ -57,3 +57,83 @@ Create a dataset with authorized cloud storage:
 .. important::
 
     The directory ``path/to/dataset`` should be empty when create an authorized storage Fusion Dataset.
+
+Import Cloud Files into Authorized Storage Dataset
+==================================================
+
+There are two methods to import cloud files into an authorized storage dataset.
+
+    - OUT-OF-THE-BOX: Import all files under a directory into a dataset.
+    - CUSTOMIZED: Use AuthData to organize cloud files into a dataset.
+
+Take the following cloud directory as an example::
+
+   datas/
+   ├── images/
+   │   ├── 00001.png
+   │   ├── 00002.png
+   │   └── ...
+   ├── labels/
+   │   ├── 00001.json
+   │   ├── 00002.json
+   │   └── ...
+   └── ...
+
+
+Out-of-the-box Method
+*********************
+
+Import all files in ``datas/images`` directory into ``train`` segment.
+
+.. code:: python
+
+   dataset_client.import_all_files("datas/images", "train")
+
+
```

Comment:
Please explain explicitly that if there is labels information, use the customized method.